hooks/test/browser/combinations.test.js

```diff
@@ -254,19 +254,27 @@ describe('combinations', () => {
 			'effect outer call <span>hello 1</span>'
 		]);
 
+		// NOTE: this order is (at the time of writing) intentionally different from
```

Comment:
We can support this if we want. I had it in my PR by doing it separately

Reply:
Yup. I think I'd like to see if its necessary though. I think we can save some bytes (and possibly runtime perf for large apps) if we only have to loop through this array once instead of twice (once to run all cleanups, again to run all effects). If someone reports an issue where they have a valid use case for React's behavior, we can look at doing the two loops again.